lib/model.js

```diff
@@ -974,6 +974,14 @@ class Model {
         this._dataTypeChanges[name] = definition.type._isChanged;
       }
 
+      // Mark array enum value for Postgres.
+      if (definition.type instanceof DataTypes.ARRAY &&
+        DataTypes.ARRAY.is(definition.type, DataTypes.ENUM)) {
+        definition.type.enumName = '"' + (this._schema || 'public')
```

Comment:
Use template strings for better readability

Reply:
Will do, wasn't sure if you wanted to keep older Node compatibility or not.